server/base/src/test/java/org/apache/accumulo/server/util/TableDiskUsageTest.java

```diff
@@ -276,36 +288,41 @@ private static String getTableName(TableId tableId) {
     return tableIdToNameMap.get(tableId);
   }
 
-  private static void appendFileMetadata(Map<Key,Value> tableEntries, TabletFile file, long size) {
-    tableEntries.put(
-        new Key(new Text(file.getTableId() + "<"),
-            MetadataSchema.TabletsSection.DataFileColumnFamily.NAME, file.getMetaInsertText()),
-        new DataFileValue(size, 1).encodeAsValue());
+  private static void appendFileMetadata(List<TabletMetadata> realTabletsMetadata,
+      StoredTabletFile file, TableId id, long size) throws Exception {
+    Map<StoredTabletFile,DataFileValue> files = Map.of(file, new DataFileValue(size, 1));
+    TabletMetadata tm = EasyMock.createMock(TabletMetadata.class);
```

Comment:
It looks like there is a convertRow method that is used to create TabletMetadata objects for testing. So you don't need to mock it with EasyMock... you can create a concrete instance of it containing test state.

In the main branch, it looks like this was changed to a convenient builder.